packages/plugin-linebreak/LineBreak.ts

```diff
@@ -17,10 +17,17 @@ export class LineBreak extends JWPlugin {
         if (context.currentNode.nodeName === 'BR') {
             const node = context.currentNode;
             const domNodes = [node];
+            const attributes = Parser.parseAttributes(context.currentNode);
             if (!node.nextSibling) {
                 // A <br/> with no siblings is there only to make its parent visible.
                 // Consume it since it was just parsed as its parent element node.
                 // TODO: do this less naively to account for formatting space.
+                attributes.forEach((value: string, name: string) => {
+                    if (!context.parentVNode.attributes) {
+                        context.parentVNode.attributes = new Map<string, string>();
+                    }
```

Comment:
I think this instantiation should be done in the node, such that we never have to worry about it, neither for writing attributes nor reading them.